$OpenBSD$
Fix from upstream:

Temporary workaround for data loss on copy/move bugs.
BUG: 281552
BUG: 279722
FIXED-IN: 4.8

Git revision b5b254a2:9d1da940
--- akonadi/pastehelper.cpp
+++ akonadi/pastehelper.cpp
@@ -137,6 +137,12 @@ KJob* PasteHelper::pasteUriList( const QMimeData* mimeData, const Collection &de
   }
 
   TransactionSequence *transaction = new TransactionSequence( session );
+
+  //FIXME: The below code disables transactions in otder to avoid data loss due to nested
+  //transactions (copy and colcopy in the server doesn't see the items retrieved into the cache and copies empty payloads).
+  //Remove once this is fixed properly, see the other FIXME comments.
+  transaction->setProperty( "transactionsDisabled", true );
+  
   switch ( action ) {
     case Qt::CopyAction:
       if ( !items.isEmpty() )
